### DISCUSSION TITLE
Can't C/P With Keyboard Shortcuts in Hydrogen 1

### DISCUSSION THREAD

**Discussion by mark761** [hydrogen-music/hydrogen#1414] [Hydrogen on Linux]
For some reason all the laptop keys I normally use for cut/copy/paste are tied to the drum sounds.  What can I do to fix this?

**Comment by cme** [comment]
Which version of Hydrogen are you using? 1.0? 1.1? 1.0 doesn't have any copy / paste functionality so Ctrl-X, Ctrl-V (or ⌘-X, ⌘-V) etc will will be tied to instruments.

(Yes, it's a bit weird that this happens even with Ctrl/⌘ is held down. We should probably change that).

1.1 was the first Hydrogen release with copy and patse functionality, and keyboard shortcuts are mapped by the Qt library so should be the same as any other application on your system. However, they'll onlyn work when the main Hydrogen window has the input focus (eg. with the mixer in the foreground instead, you'll just get the instruments on keypresses since there's nothing to copy and paste in the mixer)
